lib/rucio/core/replica.py

```diff
@@ -1789,7 +1868,8 @@ def __cleanup_after_replica_deletion(scope_name_temp_table, scope_name_temp_tabl
     while parents_to_analyze:
         did_associations_to_remove = set()
 
-        session.query(scope_name_temp_table).delete()
+        stmt = delete(scope_name_temp_table)
+        session.execute(stmt)
         session.execute(insert(scope_name_temp_table), [sn._asdict() for sn in parents_to_analyze])
```

Comment:
Break this query into statement/session.execute(statement) please

Reply:
Changed to:

```
        values = [sn._asdict() for sn in parents_to_analyze]
        stmt = insert(scope_name_temp_table)
        session.execute(stmt, values)
```